pkg/apis/kops/v1alpha1/conversion.go

```diff
@@ -0,0 +1,239 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
> will get its own conversions.go

That is all I needed to read - and we should research how k8s is pulling off the alpha shenanigans - might be useful..